data/alsa_drivers.rb

```diff
@@ -122,7 +122,11 @@ def params
 
   # find all sound drivers below the given path
   def self.find_all(path)
-    files = Dir.glob(File.join(path, "**", "snd-*.ko.xz")).select { |f| File.file?(f) }
+    files = (Dir.glob(File.join(path, "**", "snd-*.ko"))
+             + Dir.glob(File.join(path, "**", "snd-*.ko.gz"))
+             + Dir.glob(File.join(path, "**", "snd-*.ko.xz"))
+             + Dir.glob(File.join(path, "**", "snd-*.ko.zst"))
+            ).select { |f| File.file?(f) }
```

Comment:
@lslezak gave me an idea about this in the IRC channel a couple of days ago

```suggestion
    files = Dir.glob(File.join(path, "**", "snd-*.ko{,.gz,.xz,.zst}")).select { |f| File.file?(f) }
```